### DISCUSSION TITLE
Possible to pull/download some files (via rsync)?

### DISCUSSION THREAD

**Discussion by mandrasch** [deployphp/deployer#3179] [Help needed]
Hey everyone,

I just experimented with Deployer v7 and Statamic (https://dev.to/mandrasch/statamic-meets-hetzner-cloud-ploi-and-deployer-2ko5) and I was wondering:

Could I also download two file directories (from remote servers `shared/`-folder) to my local project folder? 

For Statamic I would download the media assets for example from the live server, since these are not in stored in git. 

I saw https://deployer.org/docs/7.x/contrib/rsync but only "local -> remote" is mentioned there.

Thanks in advance for suggestions and thanks for providing Deployer as Open Source! :clap:

**Comment by hansipete** [comment]
I think we're on a similar mission... just wanted to tell you, that I had a look at the source code and found a task named `download()`.

Don't know how to integrate it in my flow, but this is working for downloading content that got edited on the live server:
```php
// tasks
task('download:content', function(){
    info('Downloading content from the live server...');
    download('{{deploy_path}}/current/content/', __DIR__ . "/content/", ['progress_bar' => true]);
});
```
